config.yaml

```diff
@@ -4,6 +4,10 @@
 # See https://juju.is/docs/config for guidance.
 
 options:
+  addons:
+    description: Addons to enable
+    default: "dns rbac hostpath-storage ingress"
```

Comment:
Do we need the hostpath-storage by default given we do not want it on clusters?

Reply:
`hostpath-storage` is good enough for workloads that perform their own clustering and replication.

We can definitely revisit this default in the future, once we have a good out-of-the-box cluster storage solution.